pkgs/development/embedded/bootcommander/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchgit, cmake, libopenblt }:
+
+stdenv.mkDerivation rec {
+
+  pname = "bootcommander";
+  version = "011500";
+
+  src = fetchgit {
+    url = "https://github.com/feaser/openblt.git";
+    rev = "openblt_v${version}";
+    sparseCheckout = [ "Host/Source/" ];
+    sha256 = "sha256-cWdc6Imd6IBqYQRNEafp3bDDQLgdpE90fxSkY+vzhAM=";
+  };
+
+  preConfigure = "cd ./Host/Source/BootCommander/";
+
+  installPhase = ''
+    ls
+    cd $NIX_BUILD_TOP/openblt/Host/
+    install -D BootCommander $out/bin/BootCommander
```

Comment:
You are sure this isn't supposed to be lowercased?

Reply:
the binary output unfortunately is camel case